lib/plugins/features/movies/controller.js

```diff
@@ -34,3 +37,25 @@ exports.list = async (query) => {
 
   }).fetchAll({ withRelated: ['locations'] });
 };
+
+exports.addLocationToMovie = async (movie_id, payload) => {
+  const movie = await new Movie({ id: movie_id }).fetch({ withRelated: ['locations'] });
```

Comment:
To catch a nonexistent movie, you can alternatively add `required: true` to the `.fetch` options and then catch the no rows found error that the fetch would throw. The `!movie` check below should be fine, but just calling that out because we typically use `required: true` to handle existent checks in our code bases!

Reply:
Check out [this approach](https://github.com/shreyasj2006/sfmovies/commit/667f6f3ba084b7a3a5e9677f826c6363a345a754#diff-53570d96de57c58f9c7b4cbcb912cab7a) that takes more advantage of `require: true` for both the movie and the location.